addons/mail/static/src/xml/owl/component/attachment.xml

```diff
@@ -0,0 +1,139 @@
+                t-on-click="_onClickImage"/>
+            <div
+                class="o_Attachment_main"
+                t-if="props.hasLabelForCardLayout">
+                <div class="o_Attachment_filename">
+                    <t t-esc="storeProps.attachment.filename"/>
+                </div>
+                <div class="o_Attachment_extension">
+                    <t t-esc="env.store.getters.attachmentExtension(props.attachmentLocalId)"/>
+                </div>
+            </div>
+            <div class="o_Attachment_aside">
+                <div
+                    aria-label="Uploading"
+                    class="
+                        o_Attachment_asideItem
+                        o_Attachment_asideItemUploading"
+                    title="Uploading"
+                    t-if="
+                        storeProps.attachment.composerId and
+                        storeProps.attachment.isTemporary"
+                    t-key="'uploading'">
+                    <i
+                        class="
+                            fa
+                            fa-spin
+                            fa-spinner"/>
+                </div>
+                <div
+                    aria-label="Uploaded"
+                    class="
+                        o_Attachment_asideItem
+                        o_Attachment_asideItemUploaded"
+                    title="Uploaded"
+                    t-if="
+                        storeProps.attachment.composerId and
+                        !storeProps.attachment.isTemporary"
+                    t-key="'uploaded'">
+                    <i
+                        class="
+                            fa
+                            fa-check"/>
+                </div>
+                <div
+                    aria-label="Remove"
+                    class="
+                        o_Attachment_asideItem
+                        o_Attachment_asideItemUnlink"
+                    t-att-class="{
+                        'o-pretty': storeProps.attachment.composerId,
+                    }"
+                    title="Remove"
+                    t-if="
+                        props.isEditable and
+                        !storeProps.attachment.isTemporary"
+                    t-key="'unlink'"
+                    t-on-click="_onClickUnlink">
+                    <i
+                        class="
+                            fa
+                            fa-times"/>
+                </div>
+                <div
+                    aria-label="Download"
+                    class="
+                        o_Attachment_asideItem
+                        o_Attachment_asideItemDownload"
+                    title="Download"
+                    t-if="
+                        props.isDownloadable and
+                        !storeProps.attachment.isTemporary"
+                    t-key="'download'"
+                    t-on-click="_onClickDownload">
+                    <i
```

Comment:
generally, I think this is way too many lines for just 4 nested divs with images